interactive_engine/executor/ir/runtime/src/process/operator/map/get_v.rs

```diff
@@ -33,46 +33,28 @@ struct GetVertexOperator {
     start_tag: Option<KeyId>,
     opt: VOpt,
     alias: Option<KeyId>,
+    labels: Vec<LabelId>,
 }
 
-impl FilterMapFunction<Record, Record> for GetVertexOperator {
-    fn exec(&self, mut input: Record) -> FnResult<Option<Record>> {
-        if let Some(entry) = input.get(self.start_tag) {
-            if let Some(e) = entry.as_edge() {
-                let (id, label) = match self.opt {
-                    VOpt::Start => (e.src_id, e.get_src_label()),
-                    VOpt::End => (e.dst_id, e.get_dst_label()),
-                    VOpt::Other => (e.get_other_id(), e.get_other_label()),
-                    _ => unreachable!(),
-                };
-                let vertex = Vertex::new(id, label.map(|l| l.clone()), DynDetails::default());
-                input.append(vertex, self.alias.clone());
-                Ok(Some(input))
-            } else if let Some(graph_path) = entry.as_graph_path() {
-                // TODO: we do not check VOpt here, and we treat all cases as to get the end vertex of the path.
-                let path_end = graph_path.get_path_end().clone();
-                input.append(path_end, self.alias.clone());
-                Ok(Some(input))
+impl GetVertexOperator {
+    fn contains_label(&self, label: Option<&LabelId>) -> FnExecResult<bool> {
+        if self.labels.is_empty() {
+            // no label constraint
+            Ok(true)
+        } else {
+            if let Some(label) = label {
```

Comment:
Why we want to allow `None` label input in the function on the one hand, while throwing error while calling `None` label on the other hand? Simple pass `&LabelId` instead of `Option<&LabelId>`, no?

Reply:
This function is to test if a vertex satisfied the label condition: 
1) if without label condition, just return true (ignoring whether the label of `RuntimeVertex` is `Some(&LabelId)` or `None`);
2) if with label condition, then it would further verify the RuntimeVertex label. And actually, RuntimeVertex will always carry the label for now.

Reply:
I know, then this function should not accept Option<&LabelId>, but let the called judge.